resources-domain/resources-service/migrations/20230809143655-create-ResourcesAudits.js

```diff
@@ -0,0 +1,203 @@
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "ResourceStringAttributes_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."ResourceStringAttributes"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger ResourceStringAttributes_audit_trigger created');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "Globals_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."Globals"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger Globals_audit_trigger created');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "ResourceReferenceStringAttributeValues_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."ResourceReferenceStringAttributeValues"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger ResourceReferenceStringAttributeValues_audit_trigger created');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "ResourceBooleanAttributes_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."ResourceBooleanAttributes"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger ResourceBooleanAttributes_audit_trigger created');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "ResourceNumberAttributes_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."ResourceNumberAttributes"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger ResourceNumberAttributes_audit_trigger created');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "ResourceDateTimeAttributes_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."ResourceDateTimeAttributes"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger ResourceDateTimeAttributes_audit_trigger created');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "Accounts_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."Accounts"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger Accounts_audit_trigger created');
+
+    await queryInterface.sequelize.query(`
+      CREATE TRIGGER "ResourceReferenceStringAttributes_audit_trigger"
+      AFTER INSERT OR DELETE OR UPDATE 
+      ON resources."ResourceReferenceStringAttributes"
+      FOR EACH ROW
+      EXECUTE FUNCTION resources.audit_trigger();
+    `);
+    console.log('Trigger ResourceReferenceStringAttributes_audit_trigger created');
+  },
+
+  down: async queryInterface => {
+    await queryInterface.sequelize.query(
+      `DROP TRIGGER IF EXISTS "Resources_audit_trigger" ON resources."Resources";`,
```

Comment:
We need to drop the trigger function as part of the down script too, after the triggers but before the table